oval_graph/json_to_html.py

```diff
@@ -92,20 +92,11 @@ def create_dict_of_rule(self, rule):
         return self.create_dict_of_oval_node(self.oval_tree)
 
     def _put_to_dict_oval_trees(self, dict_oval_trees, rule, date=None):
-        dict_oval_trees[rule] = self.create_dict_of_rule(rule)
+        dict_oval_trees[rule.replace(
+            'graph-of-', '')] = self.create_dict_of_rule(rule)
```

Comment:
`graph-of-` looks like a constant, so please declare it somewhere, and don't repeat the string.